<launch>
<node pkg="motion_planning_ros" name="gridmap_ros_node" type="gridmap_ros_node" output="screen">
    <param name="local_pcd_map_topic_name" value="/map_generator/local_cloud"/>
    <param name="global_pcd_map_topic_name" value="/map_generator/global_cloud"/>
    <param name="local_occupancy_grid_map_topic_name" value="/local_occupancy_grid_map"/>
    <param name="global_occupancy_grid_map_topic_name" value="/global_occupancy_grid_map"/>

</node>
<include file="$(find map_generator)/launch/random_map_generation.launch"/>
</launch>